swifttool/manager.py

```diff
@@ -0,0 +1,216 @@
+import copy
+import logging
+import math
+import os
+import shutil
+import stat
+import subprocess
+import sys
+import tempfile
+import time
+import yaml
+from logging.handlers import SysLogHandler
+from utils import get_devices, RING_TYPES, env
+from swift.common.ring import RingBuilder
+
+
+class CapacityManager(object):
+
+    def __init__(self, iterations=10,
+                 def_file='/etc/swift/ring_definition.yml',
+                 cluster_changes=None, cluster_config=None):
+        self.iterations = iterations
+        env.key_filename = '/home/swiftops/.ssh/id_rsa'
```

Comment:
no hardcodes in a constructor.  If we want to set defaults, that is fine, but they need to be constants that can be overridden.